bugfix spiner home

volodymyr-rokhmanov opened the pull request:
Убрал действие по Esc, спинер работает

volodymyr-rokhmanov committed:
bugfix spiner home

volodymyr-rokhmanov committed:
remove EL and event

Заменил addEventListener в файле library.js на функцию, убрал ненужные эвенты там-же. Снял is-hidden с ссылки Library - все работает, но лучше проверить с регистрацией. Это функция, которая меняет цвета на кнопках.

ArtemTeslenko commented on src/js/customFunction/getBtnLibrary.js:
```diff
@@ -1,14 +1,19 @@
+  refs.watchedEl.addEventListener('click', onWatchedBtn);
+}
+
+export function onQueueBtn() {
```
Це вже ныкуди не експортуэться.

ArtemTeslenko commented on src/js/customFunction/getBtnLibrary.js:
```diff
@@ -1,14 +1,19 @@
 }
 
-export function onWatchedBtn(event) {
+export function onWatchedBtn() {
```
Те саме.

volodymyr-rokhmanov committed:
change header background

Убрал ненужные экспорты, растянул фон на весь Header и в Home, и в Library. Медиа запросы работаютю

ArtemTeslenko committed:
Merge branch 'main' into origin/bugfix-searchfilm-spiner
